### ISSUE TITLE
Turbocharging Quest scans

### ISSUE THREAD

**Issue by matt-yml** [matt-yml/pk#14]
Several customers have asked about doing quests shortly after midnight.
Rebalancing system resources from direct IV scanning to rapid Quests.
Will be important as we add AR Quests the current Normal Quest scanning.

Also improves scans for 10am event re-questing.
Rapid scans also make for multiple attempts at recovering if there are glitches overnight.

**Comment by matt-yml** [comment]
Quest scanning used to take about 6 hours and need all bots involved.
RDU is down to ~100m and other territories are under an hour now. 
Went from using 90 -> 2000 accounts (Level +30) to scan.

**Comment by matt-yml** [comment]
Need lots of level 30 accounts to do this many scans. Currently at 4044.
This will go grow as areas are added and both quests are scanned at each stop.

Doing lots of account generation and leveling up with any spare compute cycles.

**Comment by matt-yml** [comment]
Almost all territories are getting done under an hour now. 
Need a few more stable runs and declare this one done.
Move on to Alt non AR tasks after this


![Image](https://user-images.githubusercontent.com/41926716/179372628-7f12d617-66cb-4776-b630-32e4e194ad66.png)

**Comment by matt-yml** [comment]
Having issues where RDU scans ~90% and doesn't finish, but bots move onto their mon tasks.
For now I just assign a small clean up group.
This will require some heavy duty [fence splitting to find cells that stop the bootstrap](https://discord.com/channels/456432530531549195/496313484112560128/981918237996507216). 
The cartesian system now has bum tools, making it harder. 

Will address this properly when everything is converted to geoJSON files, which will happen with the new react front end.
Basically, working with bandaids till the next big switch is complete.

**Comment by matt-yml** [comment]
All major areas are completed under an hour. 
Turbo charge complete.
Will come back when alternative quests need to be scanned.

Opening a [lower level bug](https://github.com/mattlesnake/gore/issues/369) in gore repo to fix the jamming up around 90%
